packages/docs/docs/guides/brand-new.md

```diff
@@ -124,17 +137,19 @@ const tools: Record<string, Tool> = {
 
 More detail: [`UseTools` API docs](../api/modules/batteries_use_tools#usetools).
 
-## Accessing Knowledge ("Docs QA")
+### Accessing Knowledge (AKA "Docs QA")
+
+LLMs have "soft knowledge" of the world, but if you just ask a question without providing any context, they're prone to [hallucination](<https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)>). Additionally, because LLMs are only trained on public data, they don't have context on private data your or your app may care about.
```

Comment:
```suggestion
LLMs have "soft knowledge" of the world, but if you just ask a question without providing any context, they're prone to [hallucination](https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)). Additionally, because LLMs are only trained on public data, they don't have context on private data your or your app may care about.
```

Are the `<` and `>` necessary?

Reply:
apparently they are required by prettier. must be a way to have the parens in the URL not interfere with the markdown syntax for links.